tests/collection.rs

```diff
@@ -672,3 +676,501 @@ async fn test_put_rotate_journal() {
     let coll = database.drop_collection(collection_name).await;
     assert_eq!(coll.is_err(), false);
 }
+
+#[cfg(not(feature = "arango3_7"))]
+#[maybe_async::test(
+    any(feature = "reqwest_blocking"),
+    async(any(feature = "reqwest_async"), tokio::test),
+    async(any(feature = "surf_async"), async_std::test)
+)]
+async fn test_post_create_document() {
```

Comment:
I think these tests are related to document. Should we move all these document related test to `tests/document.rs`?

Reply:
I agree actually. That is a good idea :) I ' ll do it soon:)